code/modules/research/techweb/all_nodes.dm

```diff
@@ -1136,7 +1136,7 @@
 	display_name = "Basic Shuttle Research"
 	description = "Research the technology required to create and use basic shuttles."
 	prereq_ids = list("bluespace_travel", "adv_engi")
-	design_ids = list("engine_plasma", "engine_ion", "engine_heater", "engine_smes", "shuttle_helm", "rapid_shuttle_designator")
+	design_ids = list("engine_plasma", "engine_fire", "engine_ion", "engine_heater", "engine_fire_heater", "engine_smes", "shuttle_helm", "rapid_shuttle_designator")
```

Comment:
In practice you wont see this being crafted due to our RnD being restricted heavily. If you want players to be able to craft them consider adding them to cargo or as a tech disk in ruins.

Reply:
added it to cargo for 2000